app/controllers/admin/support_notices_controller.rb

```diff
@@ -0,0 +1,80 @@
+class Admin::SupportNoticesController < Admin::BaseController
+  before_action :load_support_notice, only: [:confirm_delete, :destroy, :edit, :show, :update]
+  before_action :check_inactive, only: [:confirm_delete, :destroy]
+  
+  # GET /admin/notices/support
+  def index
+    authorize(SupportNotice)
+
+    @pagy, @support_notices = pagy(SupportNotice.order(active: :desc, updated_at: :desc))
+  end
+
+  # GET /admin/notices/support/1
+  def show
+  end
+
+  # GET /admin/notices/support/new
+  def new
+    @support_notice = authorize SupportNotice.new
+  end
+
+  # GET /admin/notices/support/1/edit
+  def edit
+  end
+
+  # POST /admin/notices/support
+  def create
+    @support_notice = authorize SupportNotice.new(support_notice_params)
+
+    if @support_notice.save
+      AdminActivity.log_action(current_admin, @support_notice, action: "create", summary: @support_notice.notice)
+
+      flash[:notice] = t(".created")
```

Comment:
```suggestion
      flash[:notice] = t(".success")
```
is a bit more standard. Same goes for the other two actions